assets/Readme.md

```diff
@@ -1 +1,21 @@
-
+# Image Tour of entire Application☣️
```

Comment:
No need to write in ```assets/Readme.md``` because there is already had in ```imageTour.md``` file. You can checkout this file and got idea :)